agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/ConnectionStringAccessor.java

```diff
@@ -48,7 +47,7 @@ public void setValue(String value) {
             // now that we know the user has opted in to tracing, we need to init the propagator and sampler
             DelegatingPropagator.getInstance().setUpStandardDelegate();
             // TODO handle APPLICATIONINSIGHTS_SAMPLING_PERCENTAGE
-            DelegatingSampler.getInstance().setAlwaysOnDelegate();
+            DelegatingSampler.getInstance().setDelegate(100);
```

Comment:
for Azure Functions consumption users who don't opt in to monitoring, it's more efficient if we use `Sampler.alwaysOff()` (it will return noop spans, etc which will be more efficient than full spans, etc)